build/codegen/rook-client-python/README.md

```diff
@@ -0,0 +1,67 @@
+# rook-client-python
```

Comment:
How about if we just name this directory `python`? Everything in this dir should be about rook client code generation anyway.